### PR TITLE
Collect Sql error message (profiler only)

### PR DESCRIPTION
**Pull request by abaranch**
#101

### CODE REVIEW

File: Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs [hunk 1]
```diff
@@ -45,11 +45,10 @@ public static void Reset()
         /// </summary>
         public static void Start()
         {
-            Process process = new Process { StartInfo = StartIis };
-            process.Start();
-            process.WaitForExit();
+            Stop();
```
Comment (dnduffy) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs]: ?

**Comment by dnduffy** [comment]
Start calling Stop looks odd but if my assumption is correct then it is okay but should probably be commented and then :shipit:.

File: Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs [hunk 2]
```diff
@@ -60,7 +59,9 @@ public static void Start()
         public static void Stop()
         {
             Process process = new Process { StartInfo = StopIis };
```
Comment (dnduffy) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs]: So StartIis was removed and only StartW3Svc remains in Start but Stop only has StopIis, should there be a StopW3Svc?

File: Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs (hunk 1 above)
Comment (abaranch) [inline Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs]: Good catch. Thank you. I misread. I thought start is doing stop and than start of the same process.

---
In reply to: [73772634](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/115#discussion_r73772634) [](ancestors = 73772634)

**Pull request merged**